packages/core/src/components/Alert/Alert.scss

```diff
@@ -22,20 +22,25 @@ Markup:
 Style guide: components.alert
 */
 
-$alert-icon-size: $spacer-5;
+$h-padding: $spacer-2;
+$bar-size: $spacer-1;
```

Comment:
Nit: Could we could name these consistently (ie. namespacing w/ `$alert-`), so it's easier for people to theme and less likely to conflict with a project's existing Sass variables?